src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,56 @@
-export const Tabs = () => {};
+import { Dispatch } from 'react';
+
+interface TabsInterface {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: TabsInterface[]
+  selectedTab: TabsInterface
+  onSelect: Dispatch<TabsInterface>
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onSelect,
+}) => {
+  return (
+
+    <div>
+      <div className="tabs is-boxed">
+        <ul>
+
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={
+                tab.id === selectedTab.id
+                  ? 'is-active'
+                  : ''
+              }
```

Comment:
use [classnames](https://www.npmjs.com/package/classnames) lib to add classes conditionally

Reply:
Help ne. I dom`t understand how classnames works. How can i fix it in this task?

> use [classnames](https://www.npmjs.com/package/classnames) lib to add classes conditionally